res/parking/parking_manager.c

```diff
@@ -557,7 +558,13 @@ static int manager_park(struct mansession *s, const struct message *m)
 		ast_bridge_set_transfer_variables(chan, timeout_channel, 0);
 		ast_channel_unlock(chan);
 	}
-
+	
+	if (!ast_strlen_zero(parkingpos)) {
+		ast_channel_lock(chan);
+		pbx_builtin_setvar_helper(chan, "PARKINGEXTEN", parkingpos);
+		ast_channel_unlock(chan);
+	}
```

Comment:
You also need to document the new option in the XML documentation at line 96 of this file